packages/web/src/components/collection/components/AlbumDetailsText.tsx

```diff
@@ -0,0 +1,51 @@
+import { formatDate, formatSecondsAsText } from '@audius/common/utils'
+import { Flex, Text } from '@audius/harmony'
+
+import { useIsMobile } from 'hooks/useIsMobile'
+
+type AlbumDetailsTextProps = {
+  lastModifiedDate?: number | string
+  releaseDate: number | string
+  numTracks: number
+  duration: number | null
+}
+export const AlbumDetailsText = ({
+  duration,
+  lastModifiedDate,
+  numTracks,
+  releaseDate
+}: AlbumDetailsTextProps) => {
+  const isMobile = useIsMobile()
+  const renderAlbumDetailsText = () => {
+    const releaseAndUpdatedText = lastModifiedDate
+      ? `Released ${formatDate(`${releaseDate}`)}, Updated ${formatDate(
+          `${lastModifiedDate}`
+        )}`
+      : `Released ${formatDate(`${releaseDate}`)}`
+
+    const trackCountText = `${numTracks} tracks`
+    const durationText = duration ? `, ${formatSecondsAsText(duration)}` : ''
+    return isMobile ? (
+      <Flex direction='column' gap='xs'>
+        <div>{releaseAndUpdatedText}</div>
```

Comment:
can we use `Flex` instead of `div`?

Reply:
Actually I think `Box` would be the "appropriate" way but fair critique still!